src/commands/starship.rs

```diff
@@ -0,0 +1,13 @@
+use super::*;
+
+/// Starship Metadata
+#[derive(Parser)]
+#[clap(hide = true)]
+pub struct Args {}
+
+pub async fn command(_args: Args, _json: bool) -> Result<()> {
+    let configs = Configs::new()?;
+    let linked_project = configs.get_linked_project().await?;
+    println!("{}", serde_json::to_string(&linked_project)?);
```

Comment:
I don't understand how to use this command. Maybe add some descriptive docs?

![image](https://user-images.githubusercontent.com/3044853/222781911-a11a0f9e-b0de-4752-8702-63678e0325ca.png)

Reply:
This is a hidden command for `starship` (https://github.com/starship/starship)
It's used to get metadata for the prompt - it doesn't appear in help and is not meant for direct use by people.